### PR TITLE
Feature/start stop node in backend

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- move start/stop of nodes from frontend to backend
- removing a node in the frontend now also removes its associated data in s3 through an API extension in storage container
- removal of API running_interactive_services and replaced by projects/{project_id}/nodes API

Bonus:
- fixed an issue in director where starting the latest version of a service was failing to take the effective latest version
- fixed an issue in sidecar that the progress rabbit channel was not labeled with user/project ids, thus creating an error in the webserver listening to this very channel
- storage now allows deletion of the storage of a particular node (see reason why above)
- refactoring in frontend of snake case to camelcase of calls to the projects API
- updated docker-compose version to 1.25.0 in travis for future BUILDKIT use
- updated docker engine to latest in travis for future BUILDKIT use

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] **Runs in the swarm**
- [x] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: api/specs/webserver/v0/openapi-projects.yaml [hunk 1]
```diff
@@ -75,6 +75,19 @@ paths:
...
+      operationId: get_active_project
+      responses:
+        '200':
+          description: returns active project
+          content:
+            application/json:
+              schema:
+                $ref: './openapi-projects.yaml#/components/schemas/ProjectEnveloped'
+        default:
+          $ref: './openapi.yaml#/components/responses/DefaultErrorResponse'
```
Comment (sanderegg) [inline api/specs/webserver/v0/openapi-projects.yaml]: was just moved

File: services/web/server/src/simcore_service_webserver/projects/projects_handlers.py [hunk 2]
```diff
@@ -317,3 +315,68 @@
...
+        )
+    
+    node_details = await projects_api.get_project_node(request, project_uuid, user_id, node_uuid)
+    return {
+        'data': node_details
+    }
+
+@login_required
+async def delete_node(request: web.Request) -> web.Response:
+    # TODO: replace by decorator since it checks again authentication
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/projects/projects_handlers.py]: Again, all delete requests shall not block response to client.

File: api/specs/webserver/v0/openapi.yaml [hunk 3]
```diff
@@ -127,7 +121,13 @@ paths:
     $ref: './openapi-projects.yaml#/paths/~1projects~1{project_id}~1open'
```
Comment (pcrespov) [inline api/specs/webserver/v0/openapi.yaml]: Excellent! THX for helping to make API more consistent

File: services/web/server/src/simcore_service_webserver/projects/projects_handlers.py (hunk 2 above)
Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/projects/projects_handlers.py]: well this is using project_api which in turn should be using fire&forget (as per your comment above). in which case this solves it already.
additionally I am not sure this is a performance problem anyway, as the client is not waiting for the response of that call anyway.

**Pull request merged**